File: 07_funcoes/ex02_par_impar.py
"""
Exercicio 02
Tema: 07_funcoes
"""

def ehimpar(n):
    if n % 2 != 0:
        return 1
    else:
        return 0
    
def ehpar(n):
    if n % 2 == 0:
        return 1
    else:
        return 0

n = int(input("Digite um numero para saber se eh 'par' ou 'impar': "))

if ehpar(n) == 1:
    print(f"'{n}' eh par")
else:
    print(f"'{n}' eh impar")

File: 07_funcoes/ex02_par_impar.c
/*
Exercicio 02
Tema: 07_funcoes
*/

#include <stdio.h>

int ehimpar(int n){
    if(n % 2 != 0){
        return 1;
    } else{
        return 0;
    }
}

int ehpar(int n){
    if (n % 2 == 0){
        return 1;
    } else if(ehimpar(n) == 1){
        return 0;
    }
}


int main() {

    int n;
    printf("Digite um numero para saber se eh 'par' ou 'impar':\n");
    scanf("%d", &n);

    if (ehpar(n) == 1){
        printf("'%d' eh par\n", n);
    } else {
        printf("'%d' eh impar\n", n);
    }

    return 0;
}
